docs/superpowers/specs/2026-04-10-mobile-first-responsive-redesign.md

```
@@ -0,0 +1,304 @@
+- Knowledge Bases
+- Chatbot Config
+- Analytics
+- API Keys
+- LLM Providers
+- Sandbox
+
+#### Implementation
+
+New component: `MobileTabBar.vue`
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)